app/next-client-app/types/scanreport.ts

```diff
@@ -1,27 +1,23 @@
-interface ScanReportResult {
-  id: number;
-  dataset: string;
-  parent_dataset: string;
-  data_partner: string;
-  status: string;
-  created_at: Date;
-  hidden: boolean;
-}
 interface ScanReport {
```

Comment:
I think these types need clarifying. Like this type is not a `ScanReport` (the one called `ScanReportList` looks to be one).

I would say this interface should be a 

```
interface PaginatedResponse<T> {
  count: number;
  next: string | null;
  previous: string | null;
  results: T[];
}
```
so your response object is a `PaginatedResponse<ScanReport>` for a list of Scan Reports. But then can be used for anything that is paginated.